src/MeetingsSDKAdapter/controls/SwitchCameraControl.js

```diff
@@ -0,0 +1,54 @@
+import {
+  defer,
+  Observable,
+} from 'rxjs';
+import {map, distinctUntilChanged} from 'rxjs/operators';
+import MeetingControl from './MeetingControl';
+import {combineLatestImmediate} from '../../utils';
+/**
+ * Display options of a meeting control.
+ *
+ * @external MeetingControlDisplay
+ * @see {@link https://github.com/webex/component-adapter-interfaces/blob/master/src/MeetingsAdapter.js#L58}
+ */
+
+export default class SwitchCameraControl extends MeetingControl {
+  /**
+   * Calls the adapter switchCamera method.
```

Comment:
The action of the switch camera control.

or: What to do when the user changes the selected camera.